src/data/faq.json

```diff
@@ -87,6 +87,117 @@
                     }
                 ]
             },
+            {
+                "title": "Schnelltest",
+                "id": "rapid_antigen_test",
+                "active": true,
+                "accordion": [
+                    {
+                        "title": "Why are quick tests now integrated?",
+                        "anchor": "rat_why_integrate_rat",
+                        "active": true,
+                        "textblock": [
+                            "In the latest version of the Corona-Warn-App (2.1), the project team from Deutsche Telekom and SAP has integrated the announced rapid tests so that chains of infection can be interrupted even faster.",

```

Comment:
```suggestion
                            "In the latest version of the Corona-Warn-App (2.1), the project team from Deutsche Telekom and SAP has, as announced, integrated rapid tests so that chains of infection can be interrupted even faster.",

```